docs/tools/kompose/development.md

```diff
@@ -0,0 +1,145 @@
+---
```

Comment:
This doc is for kompose developers. We only put this kind of docs in github repo.